.github/workflows/ci.yml

```diff
@@ -22,7 +22,9 @@ jobs:
         ports:
           - 6379:6379
       influxdb:
-        image: influxdb
+        image: influxdb:alpine
+        options: >-
+          --name "influxdb"
```

Comment:
LOL! If you find out, please let us know :joy:

Reply:
Sure